src/everest/simulator/everest_to_ert.py

```diff
@@ -211,7 +218,7 @@ def _fetch_everest_jobs(ever_config: EverestConfig):
     Job = collections.namedtuple("Job", ["name", "source"])
     all_jobs = everest.jobs.script_names
     for default_job in all_jobs:
-        script = everest.jobs.fetch_script(default_job)
+        script = everest.jobs.fetch_script(default_job)  # type: ignore
```

Comment:
Is the "type ignore" because `fetch_script` is not typed? Maybe just type that, while you are at it?

Reply:
Thats fixed in another PR :)